systemtests/src/main/java/io/enmasse/systemtest/listener/JunitExecutionListener.java

```diff
@@ -24,6 +24,7 @@
 
     @Override
     public void testPlanExecutionStarted(TestPlan testPlan) {
+        java.security.Security.setProperty("networkaddress.cache.ttl" , "0");
```

Comment:
@famartinrh I'm not certain setting this here will be effective?  (Did you somehow test to make sure here is good?)   I think need to make sure it is set before any networking has occurred.  It might be safer to the system property sun.net.inetaddr.ttl (perhaps in Maven)

https://stackoverflow.com/questions/1256556/how-to-make-java-honor-the-dns-caching-timeout